README.md

```diff
@@ -94,11 +94,30 @@ $ sudo journalctl -t cc-runtime
 
 ### Enabling debug for various components
 
-The runtime, the shim (`cc-shim`), and the hypervisor all have separate debug
-options in the [configuration file](#Configuration).
+The runtime, the shim (`cc-shim`), the hypervisor and the proxy all have separate `enable_debug=` debug
```

Comment:
if we are going to call out `cc-shim`, do we want to call out `cc-proxy` as well?

Reply:
Yeah, I'll re-order the elements and mention the proxy binary name...